I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 104, in script call
    call initMonologue
  File "game/script.rpy", line 121, in script call
    call evilwizardman
  File "game/script.rpy", line 130, in script
    "Huh?"
IOError: Couldn't find file 'poof.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 104, in script call
    call initMonologue
  File "game/script.rpy", line 121, in script call
    call evilwizardman
  File "game/script.rpy", line 130, in script
    "Huh?"
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/ast.py", line 721, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/exports.py", line 1419, in say
    who(what, *args, **kwargs)
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/character.py", line 1249, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/character.py", line 901, in do_display
    **display_args)
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/character.py", line 647, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/ui.py", line 298, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/display/core.py", line 3325, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/display/core.py", line 3773, in interact_core
    renpy.audio.audio.interact()
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/audio/audio.py", line 1209, in interact
    periodic()
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/audio/audio.py", line 1162, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/audio/audio.py", line 1140, in periodic_thread_main
    periodic_pass()
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/audio/audio.py", line 1082, in periodic_pass
    c.periodic()
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/audio/audio.py", line 518, in periodic
    topf = load(filename)
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/audio/audio.py", line 69, in load
    rv = renpy.loader.load(fn)
  File "/Users/jylahbah/Downloads/renpy-7.4.11-sdk/renpy/loader.py", line 798, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'poof.mp3'.

Darwin-20.5.0-x86_64-i386-64bit
Ren'Py 7.4.11.2266
Remember Me 1.0
Sun Mar 27 15:56:59 2022
